action.yml

```diff
@@ -39,6 +42,12 @@ runs:
         until docker pull "$IMAGE_URL" 2>/dev/null; do
            sleep 10
         done
+    - name: Configure to use the test image
+      shell: bash
+      run: |
+        pwd
```

Comment:
What is the reason to include `pwd` and `ls -FGl` here?

Reply:
Debugging. I'm not sure what directory we're in here, we have two cases above based on whether we're checking out self-hosted or already in self-hosted.